lexicons/app/bsky/actor/searchActors.json

```diff
@@ -16,6 +16,11 @@
             "type": "string",
             "description": "Search query string. Syntax, phrase, boolean, and faceting is unspecified, but Lucene query syntax is recommended."
           },
+          "account": {
```

Comment:
Do we want to call this `actor` or `for` or something? Not sure if we've formalized a name for this elsewhere but I don't recall using `account` in other places for this, I might just not be super immersed in lexicons though.

Reply:
The only place we use "account" so far is for a user on a pds, who may not necessarily have a repository on the network.  I think `actor` fits but would be ambiguous.  My only issue with `for` is that it's a keyword in many languages 😆  Another option that comes to mind is `viewer`, which is the name of the field where we put viewer-specific data for many response lexicons.  If we land on "account" I wont be sad, though.

Reply:
I like `viewer`, or sticking with `account` as a second option.